test/docker-compose.yml

```diff
@@ -1,6 +1,15 @@
 version: '2'
 
 services:
+  dev:
```

Comment:
Developing in a container probably is not for everyone and I would remove that change before starting a final PR.

Reply:
This is still here btw